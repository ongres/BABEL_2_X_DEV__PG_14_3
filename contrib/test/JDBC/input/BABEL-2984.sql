grant execute on dbo.myproc to public
GO

grant exec on dbo.myproc to public
GO
