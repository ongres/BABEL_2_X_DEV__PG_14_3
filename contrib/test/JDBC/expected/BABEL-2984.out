grant execute on dbo.myproc to public
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: relation "master_dbo.myproc" does not exist)~~


grant exec on dbo.myproc to public
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: relation "master_dbo.myproc" does not exist)~~

